sentinelone.json

```diff
@@ -36,6 +36,13 @@
             "data_type": "password",
             "required": true,
             "order": 1
+        },
+        "max_containers": {
+            "description": "Maximum Number of Containers to Ingest",
+            "data_type": "numeric",
+            "value": 100,
```

Comment:
Should this be "default" rather than "value"?

Reply:
ok!

Reply:
It looks like from your screenshot it works, I've just never seen that key used before.

Reply:
i used default per your suggestion. yes, value does work too

Reply:
Looks like static tests are failing due to logging statements and min_phantom_version, but I believe we are not changing those now?